src/coreclr/jit/lower.cpp

```diff
@@ -2787,14 +2787,13 @@ GenTree* Lowering::OptimizeConstCompare(GenTree* cmp)
             // doing so would produce incorrect results (e.g. RSZ, RSH).
             //
             // The below list of handled opers is conservative but enough to handle the most common
-            // situations. In particular this include CALL, sometimes the JIT unnecessarily widens
```

Comment:
The `CALL` comment is obsolete. We now all of the aforementioned casts in morph. It is still useful to narrow a rare case of an unmanaged call, however.